<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUpper_Left"/>
   <addaction name="actionRemove_UL"/>
   <addaction name="actionUpper_Right"/>
   <addaction name="actionRemove_UR"/>
   <addaction name="actionLower_Left"/>
   <addaction name="actionRemove_LL"/>
   <addaction name="actionLower_Right"/>
   <addaction name="actionRemove_LR"/>
   <addaction name="actionEqualise"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUpper_Left">
   <property name="text">
    <string>Add UL</string>
   </property>
  </action>
  <action name="actionUpper_Right">
   <property name="text">
    <string>Add UR</string>
   </property>
  </action>
  <action name="actionLower_Left">
   <property name="text">
    <string>Add LL</string>
   </property>
  </action>
  <action name="actionLower_Right">
   <property name="text">
    <string>Add LR</string>
   </property>
  </action>
  <action name="actionEqualise">
   <property name="text">
    <string>Equalise</string>
   </property>
  </action>
  <action name="actionRemove_UL">
   <property name="text">
    <string>Remove UL</string>
   </property>
  </action>
  <action name="actionRemove_UR">
   <property name="text">
    <string>Remove UR</string>
   </property>
  </action>
  <action name="actionRemove_LL">
   <property name="text">
    <string>Remove LL</string>
   </property>
  </action>
  <action name="actionRemove_LR">
   <property name="text">
    <string>Remove LR</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionUpper_Left</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpper_Right</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLower_Left</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLower_Right</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEqualise</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>equalise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_LL</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_LR</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_UL</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_UR</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewItem()</slot>
  <slot>equalise()</slot>
  <slot>removeItem()</slot>
 </slots>
</ui>
